tests/e2e/specs/NestedScreen.js

```diff
@@ -0,0 +1,32 @@
+describe('Test Nested screen', () => {
```

Comment:
Having the word "Test" in your test title is redundant. It's already clear that this is a test. Also, the other test file is named with a `.spec.js` suffix. Can you rename this file to `NestedScreen.spec.js`?

Reply:
Fixed